python/py-jupytext/Portfile

```diff
@@ -21,14 +21,20 @@ long_description    Jupytext can save Jupyter notebooks as Markdown and R \
                     also convert these documents **into** Jupyter Notebooks, \
                     allowing you to synchronize content in both directions.
 
-checksums           rmd160  8551001bcc066e364250568f28afabd86451cdb4 \
-                    sha256  b289924af0836c5072915f4ad5a4ac9629775fa0ed90857557e740b02251fca3 \
-                    size    5000781
+checksums           rmd160  62152c16bb75bd6523b004ec96fcaa9db16cc13f \
+                    sha256  3171e3e2827cbd63556158eb1df107e6e38e38e14c74612ac6cc7a29f4215298 \
+                    size    5235055
 
-python.versions     37 38 39 310
+python.versions     37 38 39 310 311
+
+patch {
+    reinplace "s|jupyter_packaging~=0.7.9|jupyter_packaging>=0.7.9|g" \
```

Comment:
Periods in regular expressions that are intended to be interpreted as literal periods (rather than as "any character") should be escaped but if this reinplace only affects one line, then it should be a patchfile, not a reinplace.

```suggestion
    reinplace {s|jupyter_packaging~=0\.7\.9|jupyter_packaging>=0.7.9|g} \
```

CI fails when building the main port:

```
DEBUG: Executing reinplace: /usr/bin/sed s|jupyter_packaging~=0.7.9|jupyter_packaging>=0.7.9|g </opt/local/var/macports/build/_Users_runner_work_macports-ports_macports-ports_ports_python_py-jupytext/py-jupytext/work/jupytext-1.15.1/pyproject.toml >@file11
--->  Patching pyproject.toml: s|jupyter_packaging~=0.7.9|jupyter_packaging>=0.7.9|g
DEBUG: couldn't read file "/opt/local/var/macports/build/_Users_runner_work_macports-ports_macports-ports_ports_python_py-jupytext/py-jupytext/work/jupytext-1.15.1/pyproject.toml": no such file or directory
    while executing
"exec -ignorestderr -- {*}$cmdline"
Error: reinplace: couldn't read file "/opt/local/var/macports/build/_Users_runner_work_macports-ports_macports-ports_ports_python_py-jupytext/py-jupytext/work/jupytext-1.15.1/pyproject.toml": no such file or directory
```

The reinplace (or patchfile) needs to be confined to the subports. (Move it inside the existing `if {${name} ne ${subport}} {` block.)

Reply:
> Periods in regular expressions that are intended to be interpreted as literal periods (rather than as "any character") should be escaped but if this reinplace only affects one line, then it should be a patchfile, not a reinplace.

Are patchfiles preferred in general? To me, it seems easier to maintain a reinplace line than the extra file that is a patchfile.

> The reinplace (or patchfile) needs to be confined to the subports. (Move it inside the existing `if {${name} ne ${subport}} {` block.)

Ah, yes, it's been awhile since I've developed Portfiles. I'll make this fix.

Reply:
> Periods in regular expressions that are intended to be interpreted as literal periods (rather than as "any character") should be escaped but if this reinplace only affects one line, then it should be a patchfile, not a reinplace.

Are patchfiles preferred in general? To me, it seems easier to maintain a reinplace line than the extra patch file.

> The reinplace (or patchfile) needs to be confined to the subports. (Move it inside the existing `if {${name} ne ${subport}} {` block.)

Ah, yes, it's been awhile since I've developed Portfiles. I'll make this fix.